### DISCUSSION TITLE
怎么在有公网ip的云服务器上开启代理让本地电脑解锁网易云

### DISCUSSION THREAD

**Discussion by MisakaMiko** [1715173329/UnblockNeteaseMusic#601] [Q&A]
我在云服务器上直接运行exe，在本地电脑网易云上http代理商设置服务器为云服务器公网ip的地址，端口为8080，但是没有效果。（云服务器上的防火墙关闭了的）

**Comment by 1715173329** [comment]
`.\unblockneteasemusic.exe -e https://music.163.com`

**Reply by MisakaMiko** [reply]
不好意思，能解释明白一些么，我对服务器不是很了解

**Reply by 1715173329** [reply]
cmd 执行该命令即可。
-e 追加命令行参数。

**Reply by MisakaMiko** [reply]
好的，谢谢
